lib/ansible/modules/package_facts.py

```diff
@@ -384,8 +386,8 @@ def main():
 
     # start work
     global module
-    module = AnsibleModule(argument_spec=dict(manager={'type': 'list', 'default': ['auto']},
-                                              strategy={'choices': ['first', 'all'], 'default': 'first'}),
+    module = AnsibleModule(argument_spec=dict(manager={'type': 'list', 'elements': 'str', 'default': ['auto'],
```

Comment:
```suggestion
    module = AnsibleModule(argument_spec=dict(manager={'type': 'list', 'elements': 'str', 'default': ['auto'], choices: ['apt', 'auto', 'pacman', 'pkg', 'portage', 'rpm']}, 
                                              strategy={'choices': ['first', 'all'], 'default': 'first'}),
```

No need to remove `strategy`

Reply:
Oops. My bad. Changed it.

Reply:
Shouldn't choices be in '' ?